portfolio/src/main/webapp/style.css

```diff
@@ -4,6 +4,61 @@
   width: 650px;
 }
 
-#greeting-container {
+#fun-fact-container {
   margin-top: 20px;
+  line-height: 200%;
```

Comment:
Can we order the lines alphabetically? Here and below.

Reference:
https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order

Reply:
Done.